FlowCrypt/Core/Models/KeyAlgo.swift

```diff
@@ -14,3 +14,9 @@ struct KeyAlgo: Decodable {
     let bits: Int?
     let curve: String?
 }
+
+extension KeyAlgo: Equatable {
+    static func == (lhs: KeyAlgo, rhs: KeyAlgo) -> Bool {
+        lhs.algorithmId == rhs.algorithmId
```

Comment:
Should equate all fields. The curve may still be different, amount of bits may still be different, etc.